<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Main Toolbar</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomToolBarArea|Qt::TopToolBarArea</set>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFoldersButton"/>
   <addaction name="actionHelpButton"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <property name="toolTipsVisible">
     <bool>true</bool>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="actionCloseWindow"/>
   </widget>
   <widget class="QMenu" name="editMenu">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <property name="toolTipsVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QMenu" name="viewMenu">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <property name="toolTipsVisible">
     <bool>true</bool>
    </property>
    <addaction name="actionToggleStatusBar"/>
    <addaction name="actionLockToolbars"/>
   </widget>
   <widget class="QMenu" name="foldersMenu">
    <property name="title">
     <string>F&amp;olders</string>
    </property>
    <property name="toolTipsVisible">
     <bool>true</bool>
    </property>
    <addaction name="actionViewLauncherRootFolder"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <property name="toolTipsVisible">
     <bool>true</bool>
    </property>
    <addaction name="actionReportBug"/>
    <addaction name="actionAddToPATH"/>
    <addaction name="separator"/>
    <addaction name="actionMATRIX"/>
    <addaction name="actionDISCORD"/>
    <addaction name="actionREDDIT"/>
    <addaction name="separator"/>
    <addaction name="actionOpenWiki"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="editMenu"/>
   <addaction name="viewMenu"/>
   <addaction name="foldersMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <action name="actionToggleStatusBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Status Bar</string>
   </property>
  </action>
  <action name="actionLockToolbars">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Lock Toolbars</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset theme="settings">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Setti&amp;ngs...</string>
   </property>
   <property name="toolTip">
    <string>Change settings.</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
  </action>
  <action name="actionCloseWindow">
   <property name="icon">
    <iconset theme="status-bad">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Close &amp;Window</string>
   </property>
   <property name="toolTip">
    <string>Close the current window</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionViewLauncherRootFolder">
   <property name="icon">
    <iconset theme="viewfolder">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Launcher &amp;Root</string>
   </property>
   <property name="toolTip">
    <string>Open the launcher's root folder in a file browser.</string>
   </property>
  </action>
  <action name="actionReportBug">
   <property name="icon">
    <iconset theme="bug">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Report a Bug or Suggest a Feature</string>
   </property>
   <property name="toolTip">
    <string>Open the bug tracker to report a bug with %1.</string>
   </property>
  </action>
  <action name="actionDISCORD">
   <property name="icon">
    <iconset theme="discord">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Discord Guild</string>
   </property>
   <property name="toolTip">
    <string>Open %1 Discord guild.</string>
   </property>
  </action>
  <action name="actionMATRIX">
   <property name="icon">
    <iconset theme="matrix">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Matrix Space</string>
   </property>
   <property name="toolTip">
    <string>Open %1 Matrix space.</string>
   </property>
  </action>
  <action name="actionREDDIT">
   <property name="icon">
    <iconset theme="reddit-alien">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Sub&amp;reddit</string>
   </property>
   <property name="toolTip">
    <string>Open %1 subreddit.</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="about">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;About %1</string>
   </property>
   <property name="toolTip">
    <string>View information about %1.</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionAddToPATH">
   <property name="icon">
    <iconset theme="custom-commands">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Install to &amp;PATH</string>
   </property>
   <property name="toolTip">
    <string>Install a %1 symlink to /usr/local/bin</string>
   </property>
  </action>
  <action name="actionFoldersButton">
   <property name="icon">
    <iconset theme="viewfolder">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Folders</string>
   </property>
   <property name="toolTip">
    <string>Open one of the folders shared between instances.</string>
   </property>
  </action>
  <action name="actionHelpButton">
   <property name="icon">
    <iconset theme="help">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="toolTip">
    <string>Get help with %1 or Minecraft.</string>
   </property>
  </action>
  <action name="actionOpenWiki">
   <property name="icon">
    <iconset theme="help">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>%1 &amp;Help</string>
   </property>
   <property name="toolTip">
    <string>Open the %1 wiki</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
